docs/API.md

```diff
@@ -175,3 +176,25 @@ ORDER BY five_min
 ```
 Note that the above cast to TIMESTAMP converts the time to local time according
 to the server's timezone setting.
+
+---
+
+### `last()` and `first()`
+
+The `last()` and `first()` aggregates allow you to get the value of one column as orderer by another. For example, `last(temp, time)` will return the latest temperate value based on time within an aggregate group.
+
+**Required arguments**
+
+|Name|Description|
+|---|---|
+| `value` | The value to return (anyelement) |
+| `time` | The timestamp to use for comparison (TIMESTAMP/TIMESTAMPTZ or integer type)  |
```

Comment:
Just looked at this and doesn't seem easy. By the time you are inside the aggregate functions, you don't have info about what table the parameter comes from, nor do you have the row either. You can't do it with DEFAULT because those have to be constants. The only way I can see to do it is to modify the planner with special logic to replace a NULL param for this function with something else on hypertables. At that point I don't think it's worth it.